content/concepts/basics/04-search.md

```diff
@@ -23,17 +43,12 @@ The query engine is responsible for executing a query that is already compiled t
 
 The indexing engine is an implementation of an interface that defines methods for writing IndexDocuments to the index. These index documents are generic, they hold all the meta information by which we can search for content items in the repository. To learn more about how indexing works in sensenet, please visit the Field indexing article.
 
-# Usage (search box tips)
--> link to /api-docs/querying
+# Usage
+sensenet content repository is a query based system, which means every content or image you see on the portal is collected by a very sophisticated and fast query engine. As an end-user, developer or administrator you get the solution to find, filter and sort content even in a huge and constantly expanding repository.
+From version 7.1 sensenet provides the ISearchEngine, IIndexingEngine and IQueryEngine interfaces that let developers create their own query and index implementations.
```

Comment:
ezt a részt is sztem szkippelhetjük

Reply:
vhová még betennék vmi utalást arra, hogy a search result mindig átmegy permission ellenőrzésen, tehát kereséssel sem érsz el olyan content-eket, amikhez nincs elég jogosultságod.